Source/com/Communicator.h

```diff
@@ -228,6 +246,13 @@ namespace RPC {
         {
             return (_configuration);
         }
+        inline const std::vector<Environment>& Environments() const
+        {
+            return (_environments);
+        }
+        inline void Environments(const std::vector<Environment>& environments) {
```

Comment:
Maybe make this a move operator ? Or also allow this to be a move operator ?

Reply:
addressed